charts/tautulli/Chart.yaml

```diff
@@ -1,16 +1,20 @@
 apiVersion: v2
-appVersion: v2.5.4
-description: A Python based monitoring and tracking tool for Plex Media Server.
 name: tautulli
-version: 3.0.1
+description: Tautulli Chart
```

Comment:
Suggest we have a more descriptive description other than "'chart name' chart" - was there a problem with the original description?